### PR TITLE
Update bundle_es.properties

### PR DESCRIPTION
**Pull request by Volas171**
added last stuff i guess

### CODE REVIEW

**Comment by Angel-24** [comment]
Hi! Well, I think turrets and units should keep their original name, without translating it, since they're the own name of that unit / turret (Cliff, Fuse, and Arc). Also, the result look very weird translated to spanish.

**Review by Angel-24 — CHANGES_REQUESTED** [review]
I suggest to leave Fuse and Arc unstranslated, as they look better this way.

File: core/assets/bundles/bundle_es.properties [hunk 1]
```diff
@@ -1019,7 +1019,7 @@ unit.corvus.name = Corvus
 
 block.resupply-point.name = Punto de reabastecimiento
 block.parallax.name = Parallax
-block.cliff.name = Cliff
+block.cliff.name = Acantilado
```
Comment (Angel-24) [inline core/assets/bundles/bundle_es.properties:1022]: block.cliff.name = Cliff

File: core/assets/bundles/bundle_es.properties [hunk 2]
```diff
@@ -1195,11 +1195,11 @@ block.mend-projector.name = Proyector de Reparación
 block.surge-wall.name = Muro de Sobretensión
 block.surge-wall-large.name = Muro de Sobretensión grande
 block.cyclone.name = Cyclone
-block.fuse.name = Fuse
+block.fuse.name = Fusible
```
Comment (Angel-24) [inline core/assets/bundles/bundle_es.properties:1198]: block.fuse.name = Fuse

File: core/assets/bundles/bundle_es.properties [hunk 3]
```diff
@@ -1195,11 +1195,11 @@ block.mend-projector.name = Proyector de Reparación
...
 block.shock-mine.name = Mina electrizante
 block.overdrive-projector.name = Proyector de Aceleración
 block.force-projector.name = Proyector de Escudo
-block.arc.name = Arc
+block.arc.name = Arco
```
Comment (Angel-24) [inline core/assets/bundles/bundle_es.properties:1202]: block.arc.name = Arc

File: core/assets/bundles/bundle_es.properties (hunk 2 above)
Comment (Volas171) [reply core/assets/bundles/bundle_es.properties:1198]: ok

**Comment by ThePythonGuy3** [comment]
The names for the teams in spanish feel... Too odd.
1. They arent capitalized (dunno in english tho)
2. "amarillo" "abandonado" "rojo" just no...

**Review by Angel-24 — COMMENTED** [review]
> The names for the teams in spanish feel... Too odd.
> 
> 1. They arent capitalized (dunno in english tho)
> 2. "amarillo" "abandonado" "rojo" just no...

I think they're not capitalized because it would look like:  "Equipo Amarillo" instead of "Equipo amarillo". And it isn't the team's name, but the team's color.

Also Identifying teams by it's color is a good idea.

**Comment by ThePythonGuy3** [comment]
Well yes but in English they arent color identified, unless it got changed recently. (abandonado just sounds too off)

**Comment by Angel-24** [comment]
> Well yes but in English they arent color identified, unless it got changed recently. (abandonado just sounds too off)

yes, that's true.

**Comment by Angel-24** [comment]
but should we change it?

**Pull request closed without merging**